raster/r.info/main.c

```diff
@@ -348,7 +348,7 @@ int main(int argc, char **argv)
             fprintf(out, "cols=%d\n", cellhd.cols);
             
             fprintf(out, "cells=%jd\n",
-                    (grass_int64)cellhd.rows * cellhd.cols);
+                    (intmax_t)cellhd.rows * cellhd.cols);
```

Comment:
There is a reason for `grass_int64`, this type is portable and set at compile time.

Reply:
Reverted to grass_int64. Changed format specifier, now using PRId64.